scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -39,4 +43,5 @@ else
     exit 1
 fi
 
+# kam version check
```

Comment:
What's this comment indicating?

Reply:
I just add a comment line for `kam version` command

Reply:
But it's not checking the version?

Reply:
Its checking the existence of the correct kam binary in the PATH by running the version command. I can re phrase to a better comment for example ```Checking kam binary in the PATH through version command.``` ?

Reply:
How about something like this?

```bash
# assert that kam is on the path
```